app/views/subnets/import.html.erb

```diff
@@ -1,19 +1,20 @@
 <% title _("Import IPv4 subnets") %>
 
-<h2><%= _("The following IPv4 subnets have been found") %></h2>
-
+<h4 class='col-md-8'><%= _("The following IPv4 subnets have been found. Please review the details before creating them.") %></h4>
```

Comment:
The `col-md-8` adds a `float: left` to this element which causes it to overlap:

![selection_003](https://cloud.githubusercontent.com/assets/482089/17692127/131dc90c-6390-11e6-9717-309f1bfc2a3f.png)